### PR TITLE
add `monitor-ui` to `./tm-monitor`

### PR DESCRIPTION
**Pull request by nylira**
I've cleaned up and added the public cosmos network monitor to `./tm-monitor/monitor-ui`. It's currently hardcoded to only work on `gaia-3006`, but it's easy to change a couple variables to monitor any testnet. I will work on improving it further next week to add more information from the Tendermint RPC and to improve usability.

# Screenshots
monitor-node
monitor-block
monitor-nodes
monitor-blockchain

### CODE REVIEW

**Comment by nylira** [comment]
Searching for addresses, blocks, etc. is coming soon too!

File: tm-monitor/monitor-ui/src/components/AppFooter.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<template>
+  <footer class="app-footer">
+    <div class="container">
+      <div class="copyright">&copy; 2018 Interchain Foundation</div>
```
Comment (melekes) [inline tm-monitor/monitor-ui/src/components/AppFooter.vue]: not AiB Inc.?

**Comment by melekes** [comment]
> src/components/NiTabBar.vue, ..

are these custom components?

**Comment by melekes** [comment]
also, we should remove Cosmos image
and update title from `Cosmos Monitor` to something like `Tendermint Blockchain Monitor`

**Comment by melekes** [comment]
what about the part which uses `tm-monitor` for extended stats?

File: tm-monitor/monitor-ui/src/components/AppFooter.vue (hunk 1 above)
Comment (nylira) [reply tm-monitor/monitor-ui/src/components/AppFooter.vue]: Yeah, we should change it AIB.

**Comment by nylira** [comment]
Yes, those are custom components.

We can change the branding to Tendermint. But what's the reasoning? I know this this a Tendermint repo, but I wonder if we'd get people using this under the Cosmos brand. I'm fine with either.

> what about the part which uses tm-monitor for extended stats?

Can you show me what the extended parts are? I haven't used `tm-monitor` before, I only have experience with the RPC.

**Comment by melekes** [comment]
> We can change the branding to Tendermint. But what's the reasoning? I know this this a Tendermint repo, but I wonder if we'd get people using this under the Cosmos brand. I'm fine with either.

the reasoning is that monitor-ui can work with any Tendermint based chain. cosmos hub is just one product.

> Can you show me what the extended parts are?

https://github.com/tendermint/tools/tree/master/tm-monitor#rpc-ui
for some reason I thought you're using tm-monitor RPC for statistics like average block time, average tx per second...

**Comment by nylira** [comment]
We also have a copy of this repo to https://github.com/cosmos/explorer @adrianbrink.

For me, I slightly prefer having it under its own repo so we can make changes quickly without affecting the entire tools repo.

**Comment by melekes** [comment]
ok

**Pull request closed without merging**